ci/scripts/build-releases.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+source $(dirname $0)/setup-go.sh
```

Comment:
I think i'll leave this one as is because I'd like to use the same base docker image for the `pubsubbeat` code. Ideally I'd extract the image into its own repository but that'll require an official launch